core/classes/IssueTagTimelineEvent.class.php

```diff
@@ -45,6 +46,8 @@ public function __construct( $p_timestamp, $p_user_id, $p_issue_id, $p_tag_name,
 		$this->issue_id = $p_issue_id;
 		$this->tag_name = $p_tag_name;
 		$this->tag = $p_tag;
+
+		$this->tag_row = tag_get_by_name( $p_tag_name );
```

Comment:
Why don't you remove the class property and move this to the html() method rather than here?  Technically speaking skip() could return true, though we don't override it here.  Also why does tag_get_by_name() uses like to find the tag rather than equality?